pomegranate/gmm.pyx

```diff
@@ -154,64 +128,71 @@ cdef class GeneralMixtureModel( Model ):
 	cdef double* summaries_ptr
 	cdef dict keymap
 	cdef int n
-	cdef public int hmm
+	cdef public bint is_hmm_
```

Comment:
I had been putting changing this off for a bit, but it should probably be "is_vl_" for variable length. The issue isn't if it's a HMM or not, it's that the input is variable length and so needs to be handled slightly differently. Can you make that change?